cypress/integration/resources.spec.js

```diff
@@ -261,38 +263,33 @@ describe('resources', () => {
 
     cy.findByTestId(LOGIN_FORM).should('not.exist');
 
-    cy.findAllByTestId(UPVOTE_BUTTON)
-      .first()
-      .children('span')
-      .invoke('text')
-      .as('currentUpVoteCountText');
+    cy.findAllByTestId(UPVOTE_COUNT).invoke('text').as('currentUpVoteCountText');
 
     cy.findAllByTestId(UPVOTE_BUTTON).first().click({ force: true });
     cy.wait('@upvote');
 
     cy.get('@upvote').then(({ status, response }) => {
       expect(status).to.eq(200);
-      cy.get('@currentUpVoteCountText').should(
-        'not.eq',
-        `YesNumber of upvotes:${response.body.resource.upvotes}`,
-      );
+      cy.get('@currentUpVoteCountText').then(upVoteText => {
+        const upVoteNumber = parseInt(upVoteText, 10);
+        expect(response.body.resource.upvotes).to.be.oneOf([upVoteNumber + 1, upVoteNumber - 1]);
```

Comment:
wait why would it be one below or one above?

Reply:
That's because we only allow 1 vote per user. So for example, if the user has already upvoted on a resource and clicks upvote again, the vote count would decrement by one whereas if the user is yet to vote or has downvoted on that resource, upvote count would increment by one.

I am working on a PR on resource api side to include user's current vote status when fetching resources. If that get's done, we can modify this test to either assert for current vote count + 1 or -1 depending on the current vote direction.

Reply:
Sounds good.